test/apps/react-oie/src/App.js

```diff
@@ -121,16 +129,17 @@ export default function App() {
     );
   }
 
+  const topNav = (
+    <div>
+      <button onClick={startIdxFlow('authenticate')}>Login</button>
```

Comment:
shouldn't these click handlers wrap `startIdxFlow`
```javascript
<button onClick={() => startIdxFlow('authenticate')}>Login</button>
```

Reply:
startIdxFow is a higher order function which returns a function